app/controllers/compute_resources_vms_controller.rb

```diff
@@ -115,4 +115,17 @@ def run_vm_action(action)
     error _("Error - %{message}") % { :message => _(e.message) }
     redirect_to :back
   end
+
+  def load_vms
+    opts = {}
+    if @compute_resource.supports_vms_pagination?
+      return unless params['sEcho'].present? # We are not paginating, don't load vms
```

Comment:
I used it for detecting if this was a json or html call, changed to actually check the what I want to check.